.github/CODEOWNERS

```diff
@@ -2,3 +2,5 @@
 /src/snowflake/snowpark/modin/ @snowflakedb/snowpandas
 /tests/integ/modin/ @snowflakedb/snowpandas
 /tests/unit/modin/ @snowflakedb/snowpandas
+/.github/ @snowflakedb/snowpandas @snowflakedb/snowpark-python-api-reviewers
+setup.py @snowflakedb/snowpandas @snowflakedb/snowpark-python-api-reviewers
```

Comment:
can we also add `/scripts` to both teams?

Reply:
Good point, done.